Cargo.toml

```diff
@@ -15,8 +15,9 @@ name = "diffutils"
 path = "src/main.rs"
 
 [dependencies]
+chrono = "0.4.35"
 diff = "0.1.10"
-regex = "1.10.3"
+regex = "1.10.4"
```

Comment:
if used only in tests, please move it to dev-dependencies

Reply:
We are using it in `src/params.rs` https://github.com/uutils/diffutils/blob/main/src/params.rs#L3